Files/Local Files/Copy a Single file from Fileshare to SFTP/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Copy a Single file from Fileshare to SFTP",
+  "version": "1.0.0",
+  "tags": [
+    "Files",
+    "SFTP",
+    "Fileshare"
+  ],
+  "additionalSystemsUsed": [
+    "SFTP"

```

Comment:
You can remove this, as we will automatically populate this based on tasks used in the process.

Reply:
Changes made as specified